webapp/components/interop.js

```diff
@@ -989,22 +1140,56 @@ class InteropSummary extends PolymerElement {
     this.updateSummaryScores();
   }
 
+  ready() {
+    super.ready();
+    // Hide the top summary numbers if there is no investigation value.
+    if (!this.shouldDisplayInvestigationNumber()) {
+      const investigationDiv = this.shadowRoot.querySelector('#investigationSummary');
+      investigationDiv.style.display = 'none';
+      const interopDiv = this.shadowRoot.querySelector('#interopSummary');
+      interopDiv.style.display = 'none';
+      const summaryDiv = this.shadowRoot.querySelector('.summary-container');
+      summaryDiv.style.minHeight = '275px';
+    }
+  }
+
+  shouldDisplayInvestigationNumber() {
+    const scores = this.dataManager.getYearProp('investigationScores');
+    return scores !== null && scores !== undefined;
+  }
+
+  // Takes a summary number div and changes the value to match the score (with CountUp).
+  updateSummaryScore(number, score) {
+    score = Math.floor(score / 10);
+    let curScore = number.innerText;
+    new CountUp(number, score, {
+      startVal: curScore === '--' ? 0 : curScore
+    }).start();
+    const colors = this.calculateColor(score);
+    number.style.color = colors[0];
+    number.style.backgroundColor = colors[1];
+  }
+
   async updateSummaryScores() {
-    let numbers = this.shadowRoot.querySelectorAll('.summary-number');
+    const scoreNumbers = this.shadowRoot.querySelectorAll('.score-number');
     const scores = this.stable ? this.scores.stable : this.scores.experimental;
-    if (numbers.length !== scores.length) {
-      throw new Error(`Mismatched number of browsers/scores: ${numbers.length} vs. ${this.scores.length}`);
+    const summaryFeatureName = this.dataManager.getYearProp('summaryFeatureName');
+    if (!scores.length || scoreNumbers.length !== scores.length) {
```

Comment:
```suggestion
    if (!scores.length && scoreNumbers.length !== scores.length) {
```
Do you mean &&?

Reply:
This is actually from an older implementation of adding the investigation score and the first part of the logic should not be in the statement anymore. Thanks for the catch 🙂